collection/collection/src/commonTest/kotlin/androidx/collection/ScatterMapTest.kt

```diff
@@ -925,6 +925,21 @@ internal class ScatterMapTest {
         }
     }
 
+    @Test
+    fun asMapToList() {
```

Comment:
Tried to run the tests. commonTests use some not common coroutines APIs, so they don't compile for web targets. Passed for k/native and desktop.

Created a task to make them runnable for web targets too: https://youtrack.jetbrains.com/issue/COMPOSE-954